source/documentBase.py

```diff
@@ -189,7 +289,22 @@ def script_previousColumn(self, gesture):
 		self._tableMovementScriptHelper(axis="column", movement="previous")
 	# Translators: the description for the previous table column script on browseMode documents.
 	script_previousColumn.__doc__ = _("moves to the previous table column")
-
+	def script_firstRow(self, gesture):
+		self._tableMovementScriptHelper(axis="row", movement="first")
+	# Translators: the description for the first table row script on browseMode documents.
+	script_firstRow.__doc__ = _("moves to the first table row")	

```

Comment:
I don't have the background on why this would be impossible, so I don't feel I can create a meaningful issue.